common/annotations/en.xml

```diff
@@ -734,306 +734,306 @@ annotations.
+		<annotation cp="😆">closed | eyes | face | grinning | haha | hahaha | happy | laugh | lol | mouth | open | rofl | satisfied | smile | smiling | squinting</annotation>
 		<annotation cp="😆" type="tts">grinning squinting face</annotation>
-		<annotation cp="😅">cold | dejected | excited | face | grinning face with sweat | nervous | open | smile | smiling face with open mouth and cold sweat | stress | stressed | sweat</annotation>
+		<annotation cp="😅">cold | dejected | excited | face | grinning | mouth | nervous | open | smile | smiling | stress | stressed | sweat</annotation>
 		<annotation cp="😅" type="tts">grinning face with sweat</annotation>
-		<annotation cp="🤣">crying | face | floor | funny | haha | happy | hehe | hilarious | joy | laugh | laughing | laughter | lmao | lmfao | lol | lolol | lololol | rofl | roflmao | rolling | rolling on the floor laughing | rotfl | tear | tears</annotation>
+		<annotation cp="🤣">crying | face | floor | funny | haha | happy | hehe | hilarious | joy | laugh | lmao | lol | rofl | roflmao | rolling | tear</annotation>
 		<annotation cp="🤣" type="tts">rolling on the floor laughing</annotation>
-		<annotation cp="😂">crying | face | face with tears of joy | feels | funny | haha | hahaha | happy | hehe | hilarious | joy | laugh | laughing | laughter | lmao | lmfao | lol | lolol | lololol | rofl | roflmao | tear | tears</annotation>
+		<annotation cp="😂">crying | face | feels | funny | haha | happy | hehe | hilarious | joy | laugh | lmao | lol | rofl | roflmao | tear</annotation>
 		<annotation cp="😂" type="tts">face with tears of joy</annotation>
-		<annotation cp="🙂">face | happy | slightly smiling face | smile | smiling</annotation>
+		<annotation cp="🙂">face | happy | slightly | smile | smiling</annotation>
 		<annotation cp="🙂" type="tts">slightly smiling face</annotation>
 		<annotation cp="🙃">face | hehe | smile | upside-down</annotation>
 		<annotation cp="🙃" type="tts">upside-down face</annotation>
 		<!-- Generated lines from Emoji Data v14, using GenerateCldrData.java -->
-		<annotation cp="🫠">disappear | dissolve | embarrassed | haha | heat | hot | liquid | lol | melt | melting | melting face | sarcasm | sarcastic</annotation> <!-- 1FAE0 -->
+		<annotation cp="🫠">disappear | dissolve | embarrassed | face | haha | heat | hot | liquid | lol | melt | melting | sarcasm | sarcastic</annotation> <!-- 1FAE0 -->
 		<annotation cp="🫠" type="tts">melting face</annotation>
 		<annotation cp="😉">face | flirt | heartbreaker | sexy | slide | tease | wink | winking | winks</annotation>
 		<annotation cp="😉" type="tts">winking face</annotation>
-		<annotation cp="😊">blush | eye | face | glad | satisfied | smile | smiling | smiling face with smiling eyes</annotation>
+		<annotation cp="😊">blush | eye | eyes | face | glad | satisfied | smile | smiling</annotation>
```

Comment:
Not sure if this is the emoji I would expect if searching for "eye" or "eyes"?

Reply:
There's actually a bunch were it was 'adjective + eyes' where they are now separate keywords, unclear if that is ideal or not since I agree I probably wouldn't search for 'eyes' for a lot of the faces, but I may search for a type of eyes (e.g. look at the crossed out eyes further down.